pkg/prometheus/operator.go

```diff
@@ -1031,28 +968,6 @@ func (c *Operator) selectServiceMonitors(p *monitoringv1.Prometheus) (map[string
 	return res, nil
 }
 
-func (c *Operator) createTPRs() error {
```

Comment:
We're actually still going to need this until we drop 1.7 support, as on start-up of the Prometheus Operator we still want to be able to perform the migration from TPRs to CRDs and rollback in case of failure.

Reply:
Forget about this comment :) 

I'm trying to delete it, but my browser crashes every time I try to press that delete icon.